Sources/Amplitude/Plugins/iOS/UIKitUserInteractions.swift

```diff
@@ -68,13 +66,15 @@ extension UIApplication {
     @objc func amp_sendAction(_ action: Selector, to target: Any?, from sender: Any?, for event: UIEvent?) -> Bool {
         let sendActionResult = amp_sendAction(action, to: target, from: sender, for: event)
 
-        guard
-            sendActionResult,
+        guard sendActionResult,
             let view = sender as? UIView,
-            view.amp_shouldTrack(action, for: event)
+            view.amp_shouldTrack(action, for: event),
+            let actionName = NSStringFromSelector(action)
+                .components(separatedBy: ":")
+                .first
```

Comment:
What is the expected transform here? please leave a comment.

Reply:
This captures the method name of the action method and discards the argument signature from the name (i.e., `menutriggered:with:` action method becomes `menutriggered`) since we're only interested in the action the method performs.

Reply:
Please add a comment to make that clear.

This may lead to some overlap across events if we cut out all of the parameters (check out UITableViewDelegate).  It may still be the more clear option, though.

Reply:
Although including all of the parameters may add additional context around the event, I think the trend seems to be that the method name itself contains enough info about what the action method actually does. The special case of parameters defining what the action methods do does not seem to be special enough to break this pattern, in my opinion.

This also gives us a clear format for the actions we might expect from a user interaction event.

Reply:
Thanks @crleona for approving this PR! Should I go a head and merge it?

Reply:
I still wouldn't mind a comment to provide a bit more detail about the transform 😄, but that's definitely not blocking.